schema/ClaimsObject.md

```diff
@@ -0,0 +1,67 @@
+# Claims Object
+
+## What is this?
+
+The Claims Object is a JSON object used to ensure that TDF Clients
+attempting to decrypt an encrypted file under the Trusted Data Format
+has all permissions required to do so.
+
+## How does it work?
+
+When an TDF Client wishes to decrypt a file, the following steps using
+the Claims Object are made:
+
+1. The client requests a Claims Object by first authenticating via the
+OpenID Connect (OIDC) Identity Provider (IdP) with Custom Claims
+support (in this case Keycloak), and if authentication succeeds, a
+[TDF Claims Object](../schema/ClaimsObject.md) is obtained from
+Attribute Provider and signed by the IdP.  The signed OIDC JWT is
+returned to the client with the Claims Object inside. The Claims
```

Comment:
Changing all of these names and to make them true and valid in code would require us to change all of the downstream code that processes Entity Objects.  I think can do that later on after this PR.

Reply:
Should we be calling this 4.0.0 if we intend to change it soon? Maybe something like 4.0.0-a2021-06?

Reply:
> Should we be calling this 4.0.0 if we intend to change it soon? Maybe something like 4.0.0-a2021-06?

No reason to, the version string 4.0.0 has no special meaning, it will be perfectly fine to have a "first public release" which is 4.2.15 or something.

The intent is to establish the Semver standard, get people used to sticking to its rules for versioning, update the version string as per the rules when updates are made regardless of release status, and then stop worrying about the specific version number, as such.

You release versions, you don't version releases.